media-api/app/controllers/SuggestionController.scala

```diff
@@ -0,0 +1,108 @@
+package controllers
+
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.libs.json.{Reads, Json, Writes}
+import play.api.mvc.{AnyContent, Request, Controller}
+
+import com.gu.mediaservice.lib.argo.ArgoHelpers
+import com.gu.mediaservice.lib.auth
+import com.gu.mediaservice.lib.auth.{PermissionStore, KeyStore}
+
+import lib.Config
+import lib.elasticsearch.{AggregateSearchResults, ElasticSearch}
+import lib.querysyntax.Parser
+
+import scala.util.Try
+
+object SuggestionController extends Controller with ArgoHelpers {
+
+  val keyStore = new KeyStore(Config.keyStoreBucket, Config.awsCredentials)
```

Comment:
Ah yes, I forgot about that. [I've moved it over to it's own place](https://github.com/guardian/grid/commit/1884a5366ca494ac0d937f14f2cfa240f3672eef).